subsys/net/lib/lwm2m/Kconfig

```diff
@@ -358,6 +358,13 @@ config LWM2M_SWMGMT_OBJ_SUPPORT
 	help
 	  Include support for LWM2M Software Management Object (ID 9)
 
+config LWM2M_PORTFOLIO_OBJ_SUPPORT
+	bool "LwM2M Portfolio object ID support for Conformance test"
```

Comment:
No need to mention the conformance test here because it's stated in the _help_ text. To keep this aligned to other configurable object descriptions.

Reply:
Oh I forget this.